x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/rest/action/user/RestChangePasswordAction.java

```diff
@@ -46,21 +47,16 @@ public RestChangePasswordAction(Settings settings, SecurityContext securityConte
 
     @Override
     public List<Route> routes() {
-        return Collections.emptyList();
-    }
-
-    @Override
-    public List<ReplacedRoute> replacedRoutes() {
         // TODO: remove deprecated endpoint in 8.0.0
```

Comment:
Does this change satisfy this TODO ?

Reply:
5bee773347657480feb8e1cec58776d68d4943b3, 👍, yes, I agree that it does.